src/towncrier/_builder.py

```diff
@@ -17,15 +17,18 @@
 from towncrier._settings.load import Config
 
 
-# Returns issue, category and counter or (None, None, None) if the basename
-# could not be parsed or doesn't contain a valid category.
+# Returns issue, category and counter. But if the basename could not be parsed
+# or doesn't contain a valid category, returns (None, None, None) or raises
+# a ValueError if in strict mode.
 def parse_newfragment_basename(
```

Comment:
I think that this function should be kept as it is.

We can have a separate helpet method that raises an exception if the invalid placehoder is returned.